app/models.py

```diff
@@ -82,10 +117,24 @@ def get_if_owned_by(cls, ident, user, options=[]):
         return obj
 
     def is_owned_by(self, user):
+        """Return a boolean indicating whether a User has read access to this
+        object.
+
+        Parameters
+        ----------
+        user : `baselayer.app.models.User`
```

Comment:
`user` can also be a `Token` instance, I believe

Reply:
fixed